backends/advanced/src/advanced_omi_backend/routers/modules/health_routes.py

```
@@ -83,6 +85,21 @@ async def auth_health_check():
+    try:
+        _cfg = _load_root_config() or {}
+        _defaults = _cfg.get("defaults", {}) or {}
+        _models = _cfg.get("models", []) or []
+        _llm_name = _defaults.get("llm")
+        _stt_name = _defaults.get("stt")
+        _llm_def = next((m for m in _models if m.get("name") == _llm_name), None)
+        _stt_def = next((m for m in _models if m.get("name") == _stt_name), None)
+        _llm_provider = (_llm_def.get("model_provider") if _llm_def else None) or "openai"
+        _llm_model = str(_resolve_value(_llm_def.get("model_name", ""))) if _llm_def else None
+        _llm_base_url = str(_resolve_value(_llm_def.get("model_url", ""))) if _llm_def else None
+    except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)